openpype/tools/publisher/control.py

```diff
@@ -2392,8 +2392,18 @@ def _publish_iterator(self):
             ):
                 yield MainThreadItem(self.stop_publish)
 
+            # Ignore base plugin classes
+            # - 'pyblish' does not ignore them on discovery
+            if plugin in (
+                pyblish.api.Plugin,
+                pyblish.api.ContextPlugin,
+                pyblish.api.InstancePlugin,
+            ):
```

Comment:
For performance this might be more optimal as a predefined set out of the loop?

Reply:
Moved to discovery...